charts/generic-service/templates/alerts-logs.yaml

```
@@ -1,4 +1,29 @@
 {{- if and .Values.alerting.enabled .Values.alerting.logs.levels }}
+{{- if .Values.alerting.logs.lokiSideCar }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "generic-service.fullname" . }}
```

This will conflict with the `ConfigMap` created by `templates/configmap.yaml`.

```suggestion
  name: '{{ include "generic-service.fullname" . }}-loki'
```